### PR TITLE
MGDCTRS-882: fix:Show the boolean value on the review Page

### PR DESCRIPTION
**Pull request by indraraj**
closes https://issues.redhat.com/browse/MGDCTRS-882

### CODE REVIEW

File: src/app/pages/CreateConnectorPage/StepReview.tsx [hunk 1]
```diff
@@ -284,14 +281,16 @@ export const StepReviewComponent: React.FC<StepReviewComponentProps> = ({
                       <strong>{_.startCase(el)}</strong>
                     </GridItem>
                     <GridItem span={8}>
-                      {dataToHide.includes(el) ? (
-                        maskValue(modifiedObject[el])
-                      ) : typeof modifiedObject[el] === 'object' ? (
+                      {typeof modifiedObject[el] === 'object' ? (
```
Comment (gashcrumb) [inline src/app/pages/CreateConnectorPage/StepReview.tsx:284]: I wonder if pulling this into a function would make it easier to understand.  Then you could maybe do a `switch (typeof modifiedObject[el]` which would simplify at least one ternary operator.

Comment (indraraj) [reply src/app/pages/CreateConnectorPage/StepReview.tsx:284]: sure

File: src/app/pages/CreateConnectorPage/StepReview.tsx [hunk 2]
```diff
@@ -57,6 +57,27 @@ export function Review() {
   );
 }
 
+const ConnectorProperties = (
```
Comment (gashcrumb) [inline src/app/pages/CreateConnectorPage/StepReview.tsx:60]: Cool!  Can't decide if it's big enough to warrant it's own component.  I guess if there's a need to add another case or more logic around this, I would definitely suggest at that point turning this into a component to help simplify the logic more.

**Review by gashcrumb — APPROVED** [review]
:+1:  works here

File: src/app/pages/CreateConnectorPage/StepReview.tsx (hunk 2 above)
Comment (gashcrumb) [reply src/app/pages/CreateConnectorPage/StepReview.tsx:60]: That's a great idea, thanks!

**Pull request merged**